src/server/services/migrateDB.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const readline = require('readline');
+const { log } = require('../log');
+const { findMaxSemanticVersion } = require('../util');
+const { printMigrationList, migrateAll, getUniqueKeyOfMigrationList } = require('../migrations/migrateDatabase');
+const migrationList = require('../migrations/registerMigration');
+
+const rl = readline.createInterface({
+	input: process.stdin,
+	output: process.stdout
+});
+
+function findMaxVersion(list) {
+	return findMaxSemanticVersion(getUniqueKeyOfMigrationList(list));
+}
+
+function askUpdateToMax() {
+	return new Promise((resolve, reject) => {
+		rl.question('Do you want to update to the max version? [yes/no]: ', answer => {
+			if (answer === 'yes' || answer === 'no') {
+				resolve(answer);
+			} else {
+				reject(answer);
+			}
+		});
+	});
+}
+
+function askToVersion() {
+	return new Promise(resolve => {
+		rl.question('To Version: ', toVersion => {
+			resolve(toVersion);
+		});
+	});
+}
+
+function terminateReadline(message) {
```

Comment:
This file, especially this method, is very similar to the createUser script. I have an open PR to refactor out some of the methods into multiple files. I recommend doing something similar (e.g. waiting for that PR to be merged and then refactoring, etc.)